src/main/java/pro/sky/Examiner/service/JavaQuestionService.java

```diff
@@ -0,0 +1,50 @@
+package pro.sky.Examiner.service;
+
+import org.springframework.stereotype.Service;
+import pro.sky.Examiner.domain.Question;
+
+import java.util.*;
+
+@Service
+public class JavaQuestionService implements QuestionService {
+    private final Set<Question> questions = new HashSet<>();
+
+    public JavaQuestionService() {
+        questions.add(new Question("вопрос_01", "ответ_01"));
+        questions.add(new Question("вопрос_02", "ответ_02"));
+        questions.add(new Question("вопрос_03", "ответ_03"));
+    }
+
+    @Override
+    public Question add(String question, String answer) {
+        Question value = new Question(question, answer);
+        questions.add(value);
+        return value;
```

Comment:
дублирование кода, нужно просто сделать делегацию
```suggestion
        return add(new Question(question, answer));
```